web/src/app/users/UserContactMethodCreateDialog.js

```diff
@@ -60,8 +100,7 @@ export default function UserContactMethodCreateDialog(props) {
       loading={loading}
       errors={nonFieldErrors(error)}
       onClose={props.onClose}
-      // wrapped to prevent event from passing into createCM
-      onSubmit={() => createCM()}
+      onSubmit={createCM}
```

Comment:
This needs to be wrapped so we don't pass the event as an argument to `createCM`, keep the comment